docs/user-guide/01-blueprint-reference.md

```diff
@@ -629,11 +629,9 @@ The service names are systemd service units. You may specify any systemd unit fi
 
 ### Files and directories 🔵 🟤 {#files-and-directories}
 
-You can use blueprint customizations to create custom files and directories in the image. These customizations are currently restricted only to the `/etc` directory.
+You can use blueprint customizations to create custom files and directories in the image. When using the custom files and directories customization, the following rules apply:
 
-When using the custom files and directories customization, the following rules apply:
-
-- The path must be an absolute path and must be under `/etc` or `/root`.
+- The path must be an absolute path and must be under `/etc`, `/root` or `/usr/local`.
```

Comment:
Note that this is wrong, the current policy ONLY custom files (no dirs) and ONLY under `/usr/local/bin` and `/usr/local/sbin`. @lzap This might be worth fixing.

Note that https://github.com/osbuild/images/pull/540 is currently open which might change the policies once again.

Reply:
Sure: https://github.com/osbuild/osbuild.github.io/pull/63